packages/amplify-provider-awscloudformation/src/admin-login.ts

```diff
@@ -27,11 +26,14 @@ export const adminLoginFlow = async (context: $TSContext, appId: string, envName
   }
 
   const url = adminVerifyUrl(appId, envName, region);
-  printer.info(`Opening link: ${url}`);
-  await open(url, { wait: false }).catch(e => {
-    printer.error(`Failed to open web browser: ${e?.message || e}`);
+  const spinner = new AmplifySpinner();
+
+  await open(url, { wait: false }).then(() => {
+    printer.info(`Opening link: ${url}`);
+    spinner.start('Confirm login in the browser or manually paste in your CLI login key:\n');
+  }).catch(() => {
+    spinner.start('Attempt to open browser failed, please manually paste in your CLI login key:\n');
```

Comment:
I don't really want the context of this erroring out to get lost, so curious if folks think this should surface to the user as a warning (to later be used for metrics) or as a debug mode output (which I believe ends up in the log dump).

Reply:
I'm in favor of this being a warning if that is easier to capture. Maybe we can split it up into a warning and a spinner? Something like:

```suggestion
    printer.warn(`Failed to open link: ${url}`);
    spinner.start('Visit the URL above and paste in your CLI login key:\n');
```

Reply:
I don't want to `warn` the user because this is an expected and supported workflow, and there's really no action item. If anything, it would be more informative with `info` - I can do that?